### ISSUE TITLE
dashboard: create a rest api for training data

### ISSUE THREAD

**Issue by hugolgst** [olivia-ai/olivia#68]
Create a REST API to get chatbot's data for the Dashboard

- [x]  Get the number of layers
- [x]  Training data:
  - [x]  Time to train 
  - [x]  Learning rate
  - [x]  Error loss
